Providers/Modules/NPM/Plugin/plugin/npmd_config_lib.rb

```diff
@@ -309,7 +317,7 @@ def self.createEpmElements(epmHash)
                     _ruleHash["IngestionWorkspaceId"] = _iRule.has_key?("IngestionWorkspaceId") ? _iRule["IngestionWorkspaceId"] : String.new
                     _ruleHash["WorkspaceAlias"] = _iRule.has_key?("WorkspaceAlias") ? _iRule["WorkspaceAlias"] : String.new
                     _ruleHash["Redirect"] = "false"
-                    _ruleHash["WorkspaceResourceId"] = _iRule["WorkspaceResourceID"];
+                    _ruleHash["WorkspaceResourceID"] = _iRule["WorkspaceResourceID"]
```

Comment:
Can you please check  that is it "WorkspaceResourceId" or "WorkspaceResourceID" ? Because in config, the variable is set as "WorkspaceResourceId".

Reply:
Its small: WorkspaceResourceId